src/components/IntegrationTests/ScheduledJobsTab.tsx

```diff
@@ -0,0 +1,730 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import {
+  Bullseye,
+  Spinner,
+  Button,
+  EmptyState,
+  EmptyStateBody,
+  Modal,
+  Form,
+  FormGroup,
+  TextInput,
+  Checkbox,
+  Alert,
+  AlertVariant,
+  AlertGroup,
+} from '@patternfly/react-core';
+import { ActionsColumn, IAction } from '@patternfly/react-table';
+import cronstrue from 'cronstrue';
+import { useIntegrationTestScenarios } from '../../hooks/useIntegrationTestScenarios';
+import { k8sPatchResource } from '../../k8s/k8s-fetch';
+import { CronJobModel } from '../../models';
+import { RouterParams } from '../../routes/utils';
+import { Table, TableData, RowFunctionArgs } from '../../shared';
+import { useNamespace } from '../../shared/providers/Namespace';
+import { CronJob } from '../../types/cronjob';
+import {
+  createPeriodicIntegrationTestCronJob,
+  deletePeriodicIntegrationTestCronJob,
+  listPeriodicIntegrationTestCronJobs,
+} from '../../utils/create-utils';
+import { validateCronExpression, sanitizeCronInput } from '../../utils/cron-validation';
+import { useAccessReviewForModel } from '../../utils/rbac';
+import { BaseTextFilterToolbar } from '../Filter/toolbars/BaseTextFIlterToolbar';
+
+export const ScheduledJobsTab: React.FC = () => {
+  const namespace = useNamespace();
+  const { applicationName } = useParams<RouterParams>();
+  const [integrationTests, integrationTestsLoaded] = useIntegrationTestScenarios(
```

Comment:
errors are not tracked here. If the hook meets errors, the page would be crashed directly.
I noticed in the konflux-ui main branch, we have several codes in this way and all of them have the same potential.
Based on the current situation, you do not have to consider the possible error in your PR but better to do.